Tests/web3swiftTests/localTests/EIP1559Tests.swift

```diff
@@ -0,0 +1,152 @@
+import XCTest
+import BigInt
+
+@testable
+import web3swift
+
+class EIP1559Tests: XCTestCase {
+    let web3 = Web3()
+
+    let uselessBlockPart = (
+        number: BigUInt(12_965_000),
+        hash: Data(from: "0xef95f2f1ed3ca60b048b4bf67cde2195961e0bba6f70bcbea9a2c4e133e34b46")!, // "hash":
+        parentHash: Data(from: "0x2302e1c0b972d00932deb5dab9eb2982f570597d9d42504c05d9c2147eaf9c88")!, // "parentHash":
+        nonce: Data(from: "0xfb6e1a62d119228b"), // "nonce":
+        sha3Uncles: Data(from: "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347")!, // "sha3Uncles":
+        receiptsRoot: Data(from: "0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347")!, // "receiptsRoot":
+        logsBloom: EthereumBloomFilter(Data(from: "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000")!), // "logsBloom":
+        transactionsRoot: Data(from: "0x3a1b03875115b79539e5bd33fb00d8f7b7cd61929d5a3c574f507b8acf415bee")!, // "transactionsRoot":
+        stateRoot: Data(from: "0xf1133199d44695dfa8fd1bcfe424d82854b5cebef75bddd7e40ea94cda515bcb")!, // "stateRoot":
+        miner: EthereumAddress( Data(from: "0x8888f1f195afa192cfee860698584c030f4c9db1")!)!, // "miner":
+        difficulty: BigUInt(21345678965432), // "difficulty":
+        totalDifficulty: BigUInt(324567845321), // "totalDifficulty":
+        size: BigUInt(616), // "size":
+        extraData: Data(from: "0x")!, // extraData":
+        gasLimit: BigUInt(3141592), // "gasLimit":
+        gasUsed: BigUInt(21662), // "gasUsed":
+        timestamp: Date(), // "timestamp":
+        transactions: [TransactionInBlock](), // "transactions":
+        uncles: [Data]() // "uncles":
```

Comment:
More curiosity: `// "hash":` and similar comments have some specific purpose?

Reply:
see #510